src/WebJobs.Script/CloudConfig/CloudConstants.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.CloudConfig
```

Comment:
nit: rather than adding a new CloudConfig directory for these two types, can you just put them peer to ScriptConstants in the root dir?